### PR TITLE
Fix python api examples, test=develop

### PR DESCRIPTION
**Pull request by baiyfbupt**
fix some api examples:

1. paddle.fluid.layers.fsp_matrix
fsp_matrix

2. paddle.fluid.layers.im2sequence
im2sequence

3. paddle.fluid.layers.mean_iou
mean_iou

4. paddle.fluid.layers.pad2d
pad2d

5. paddle.fluid.layers.warpctc
warpctc

6. paddle.fluid.layers.polygon_box_transform
polygon_box_transform

7. paddle.fluid.layers.generate_proposal_labels
generate_proposal_labels

8. paddle.fluid.layers.generate_proposals
generate_proposals

9. paddle.fluid.layers.rpn_target_assign
rpn_target_assign

10. paddle.fluid.layers.affine_channel
affine_channel

test=develop

### CODE REVIEW

**Comment by qingqing01** [comment]
@baiyfbupt You also need to update API.spec

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -10000,6 +10008,18 @@ def affine_channel(x,
         name (str, default None): The name of this layer.
         act (str, default None): Activation to be applied to the output of this layer.
 
+
+    Examples:
+        .. code-block:: python
```
Comment (shanyi15) [inline python/paddle/fluid/layers/nn.py:10252]: please leave a blank line after `.. code-block:: python`

**Comment by shanyi15** [comment]
please test first and preview.
can refer to https://github.com/PaddlePaddle/Paddle/pull/17212

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (baiyfbupt) [reply python/paddle/fluid/layers/nn.py:10252]: Done, thanks

**Comment by baiyfbupt** [comment]
test=document_preview test=develop

**Review by qingqing01 — DISMISSED** [review]

**Review by shanyi15 — DISMISSED** [review]

**Review by qingqing01 — APPROVED** [review]

**Review by shanyi15 — APPROVED** [review]

**Pull request merged**